identity/roles.go

```diff
@@ -189,3 +190,81 @@ func (c *RolesClient) Delete(ctx context.Context, input *DeleteRoleInput) error
 
 	return nil
 }
+
+// RoleTags
+type RoleTags struct {
+	// Path to the resource.
+	Name string `json:"name"`
+	// The list of role tags assigned to this resource.
+	RoleTags []string `json:"role-tag"`
+}
+
+// GetRoleTagsInput
+type GetRoleTagsInput struct {
+	// A type of the resource. Required.
+	ResourceType string
+	// An unique identifier for a resource. Optional.
+	ResourceID string
+}
+
+// GetRoleTags
+func (c *RolesClient) GetRoleTags(ctx context.Context, input *GetRoleTagsInput) (*RoleTags, error) {
+	fullPath := path.Join("/", c.client.AccountName, input.ResourceType, input.ResourceID)
+	reqInputs := client.RequestInput{
+		Method: http.MethodGet,
+		Path:   fullPath,
+	}
+
+	response, err := c.client.ExecuteRequestRaw(ctx, reqInputs)
+	if response == nil {
+		return nil, errors.Wrap(err, "unable to get role tags")
+	}
+	if response.Body != nil {
+		defer response.Body.Close()
+	}
+
+	roleTags := response.Header.Get("Role-Tag")
+
+	result := &RoleTags{
+		Name:     fullPath,
+		RoleTags: strings.Split(roleTags, ","),
+	}
+
+	return result, nil
+}
+
+// SetRoleTagsInput
```

Comment:
comment on exported type SetRoleTagsInput should be of the form "SetRoleTagsInput ..." (with optional leading article)